### ISSUE TITLE
GCC version

### ISSUE THREAD

**Issue by bishwamittra** [fbacchus/MaxHS#4]
I tried to compile maxHS in server. In my server I have GCC5.1 . I got some compilation error due to dependency on c++11. But in my ubuntu machine, there was no problem in compiling (GCC version is 5.4). I think you need to look into this factor and probably add README about minimum GCC requirement.

**Comment by fbacchus** [comment]
Yes I've used gcc 5.4 with success. Thanks for the heads up about 5.1, and I'll update the README.
